imls-backend/DATABASE_NOTES.md

```diff
@@ -0,0 +1,112 @@
+# Notes on Working with the Database
+
+## Connect to the DB via CLI
```

Comment:
Do these instructions assume you already have docker up and running?

Reply:
I just checked in a pointer back to the backend README at the top of DATABASE_NOTES